libcloudforensics/gcp_test.py

```diff
@@ -16,28 +16,343 @@
 
 from __future__ import unicode_literals
 
+import binascii
 import unittest
 
+from googleapiclient.errors import HttpError
+
 from libcloudforensics import gcp
 
 import mock
 import six
 
+FAKE_ANALYSIS_PROJECT = gcp.GoogleCloudProject(
+  'test-target-project',
+  'fake-zone'
+)
+FAKE_ANALYSIS_VM = gcp.GoogleComputeInstance(
+  FAKE_ANALYSIS_PROJECT,
+  'fake-zone',
+  'fake-analysis-vm'
+)
+FAKE_INSTANCE = gcp.GoogleComputeInstance(
+  FAKE_ANALYSIS_PROJECT,
+  'fake-zone',
+  'fake-instance'
+)
+FAKE_DISK = gcp.GoogleComputeDisk(
+  FAKE_ANALYSIS_PROJECT,
+  'fake-zone',
+  'disk1'
+)
+FAKE_BOOT_DISK = gcp.GoogleComputeDisk(
+  FAKE_ANALYSIS_PROJECT,
+  'fake-zone-other',
+  'boot-disk'
+)
+FAKE_SNAPSHOT = gcp.GoogleComputeSnapshot(
+  FAKE_DISK,
+  'snapshot1'
+)
+FAKE_DISK_COPY = gcp.GoogleComputeDisk(
+  FAKE_ANALYSIS_PROJECT,
+  'fake-zone',
+  'disk1-copy'
+)
+
 
 class GoogleCloudProjectTest(unittest.TestCase):
   """Test Google Cloud project class."""
 
   def setUp(self):
     super(GoogleCloudProjectTest, self).setUp()
-    self.project = gcp.GoogleCloudProject('test-project')
-    self.project.GceApi = mock.MagicMock()
-    self.project.GceOperation = mock.MagicMock()
 
-  def testFormatLogMessage(self):
-    """Test formatting log message."""
+  def test_format_log_message(self):
+    """Test formatting log message. """
```

Comment:
I'm not sure why a space was added after the `.`, but the style guide recommends ending on-line docstrings by periods. (with no trailing spaces). I think there are some other places where this happens.

Reply:
Addressed, other cases as well.